ci/cpu/prebuild.sh

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 
-# Copyright (c) 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020-2022, NVIDIA CORPORATION.
 set -e
 
 #Always upload cudf packages
 export UPLOAD_CUDF=1
 export UPLOAD_LIBCUDF=1
 export UPLOAD_CUDF_KAFKA=1
-export UPLOAD_LIBCUDF_KAFKA=1
```

Comment:
I'm a bit confused about what has happened to the libcudf_kafka artifacts since they seem to be removed everywhere in this PR (ci/cpu/upload.sh, ci/cpu/build.sh) without any replacement. Are they getting built into the libcudf library somehow? Are they just no longer necessary to upload during CI builds, and we only want to upload the Python cudf_kafka? I'm not very familiar with how that's supposed to work.

Reply:
Can't say anything about this particular line (maybe AJ can comment), but the idea behind this PR is to consolidate all of the packages into one recipe (`cudf`) and produce multiple packages there. If we look at that recipe, `libcudf_kafka` is still produced as an output.

Reply:
John's statement is correct. I've consolidated the C++ packages into a single `libcudf-split` recipe, so the `libcudf_kafka` package does still exist, but it's no longer in it's own separate recipe which is why this `libcudf_kafka` specific environment variable can be removed.

Reply:
Ah I see, the splitting is being done only for the C++ packages at this stage. That clears this up then, thanks.